source/guide/3.2/guide-manager-watchdog.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: bt_wiki
+title: Cloudify Manager Watchdog Guide
+category: Guides
+publish: true
+abstract: "Guide for Cloudify Manager Watchdog"
+pageord: 900
+---
+{%summary%}{{page.abstract}}{%endsummary%}
+
+
+
+{%note title=Note%}
+The Cloudify manager watchdog is a premium feature, available only in the commercial version of Cloudify.
+{%endnote%}
+
+
+# Overview
+
+The Cloudify manager watchdog is a long-running Python process which monitors a Cloudify manager, and attempts to revive it if it goes down or is unresponsive.
+
+
+# Usage
+
+The manager watchdog may run in any environment which contains the requirements for performing recovery for the manager - meaning that it requires connectivity to the manager machine and the private key file used to connect to the manager's host.
```

Comment:
the private key file part is kind of vague. its not very clear why would i need the private key of a machine that is going to die. worth mentioning that the key will actually be used for connecting to the new machine